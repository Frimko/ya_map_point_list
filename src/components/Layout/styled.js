import styled from 'react-emotion';

export const LayoutPage = styled('div')`
    display: flex;
    flex-direction: row;
  padding: 0 24px;
  width: 100%;
  justify-content: center;
`;

export const Container = styled('div')`
    height: 500px;
    display: flex;
    flex-direction: row;
`;

export const Col1 = styled('div')`
      width: 400px;
      padding: 0 24px 24px;
      background: #fff; 
      padding: 24px;
      margin: 10px; 
      minHeight: 280;
`;

export const Col2 = styled('div')`
      width: 500px;
      padding: 0 24px 24px;
      display: flex;
      background: #fff; 
      padding: 24px;
      margin: 10px; 
      minHeight: 280
`;
