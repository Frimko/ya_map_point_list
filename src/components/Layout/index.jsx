import React from 'react';
import YandexMap from 'components/YandexMap';
import DraggableListPlacemark from 'containers/DraggableListPlacemark';
import { LayoutPage, Container, Col1, Col2 } from './styled';

const Layout = () => {
  return (
    <LayoutPage>
      <Container>
        <Col1>
          <DraggableListPlacemark/>
        </Col1>
        <Col2>
          <YandexMap/>
        </Col2>
      </Container>
    </LayoutPage>
  );
};


export default Layout;
