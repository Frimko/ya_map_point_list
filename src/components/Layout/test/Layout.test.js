import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../index';

describe('Layout component ', () => {
    let container;

    beforeEach(() => {
      container = shallow(<Layout/>);
    });

    test('snapshot', () => {
      expect(container).toMatchSnapshot();
    });
});
