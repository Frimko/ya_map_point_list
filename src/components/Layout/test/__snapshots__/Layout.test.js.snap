// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component  snapshot 1`] = `
<Styled(div)>
  <Styled(div)>
    <Styled(div)>
      <Connect(DraggableListPlacemark) />
    </Styled(div)>
    <Styled(div)>
      <YandexMap />
    </Styled(div)>
  </Styled(div)>
</Styled(div)>
`;
